packages/hadron-document/src/document.ts

```diff
@@ -386,6 +397,26 @@ export class Document extends EventEmitter {
         : this.generateObject();
     return objectToIdiomaticEJSON(obj, options);
   }
+
+  /**
+   * Expands a document by expanding all of its fields
+   */
+  expandAllFields(): void {
```

Comment:
Super nit, but we expand elements, not fields, I think it's it's just `expandAll` / `collapseAll` that'd be also fine